addon/components/flexberry-colorpicker.js

```diff
@@ -0,0 +1,191 @@
+  @property {String} flexberryClassNames.wrapper Component's wrapping <div> CSS-class name ('flexberry-colorpicker').
+  @readonly
+  @static
+
+  @for FlexberryColorpickerComponent
+*/
+const flexberryClassNamesPrefix = 'flexberry-colorpicker';
+const flexberryClassNames = {
+  prefix: flexberryClassNamesPrefix,
+  wrapper: flexberryClassNamesPrefix,
+  input: flexberryClassNamesPrefix + '-input'
+};
+
+/**
+  Flexberry colorpicker component based on jquery-minicolors plugin (see http://labs.abeautifulsite.net/jquery-minicolors).
+
+  Usage:
+  templates/my-form.hbs
+  ```handlebars
+  {{flexberry-colorpicker
+    value=model.color
+    change=(action "onModelColorChange")
+  }}
+  ```
+
+  controllers/my-form.js
+  ```javascript
+  actions: {
+    onModelColorChange(e) {
+      // Set new value to model's 'color' property.
+      this.set('model.color', e.newValue);
+
+      // Log original jQuery 'change' event object.
+      console.log(e.originalEvent);
+    }
+  }
+  ```
+
+  Usage with {{#crossLink "FlexberryDdauCheckboxActionsHandlerMixin"}}flexberry-ddau-checkbox-actions-handler mixin{{/crossLink}}:
+  ```handlebars
+  {{flexberry-colorpicker
+    value=model.color
+    change=(action "onColorpickerChange" "model.color")
+  }}
+  ```
+
+  controllers/my-form.js
+  ```javascript
+  import Ember from 'ember';
+  import FlexberryColorpickerActionsHandlerMixin from 'ember-flexberry-gis/mixins/flexberry-colorpicker-actions-handler';
+
+  export default Ember.Controller.extend(FlexberryColorpickerActionsHandlerMixin, {
+  });
+  ```
+
+  @class FlexberryColorpickerComponent
+  @extends <a href="http://emberjs.com/api/classes/Ember.Component.html">Ember.Component</a>
+  @uses RequiredActionsMixin
+  @uses DomActionsMixin
+  @uses DynamicActionsMixin
+  @uses DynamicPropertiesMixin
+*/
+let FlexberryColorpickerComponent = Ember.Component.extend(
+  RequiredActionsMixin,
+  DomActionsMixin,
+  DynamicActionsMixin,
+  DynamicPropertiesMixin, {
+
+    /**
+      Component's required actions names.
+      For actions enumerated in this array an assertion exceptions will be thrown,
+      if actions handlers are not defined for them.
+
+      @property _requiredActions
+      @type String[]
+      @default ['change']
+    */
+    _requiredActionNames: ['change'],
+
+    /**
+      Reference to component's template.
+    */
+    layout,
+
+    /**
+      Reference to component's CSS-classes names.
+      Must be also a component's instance property to be available from component's .hbs template.
+    */
+    flexberryClassNames,
+
+    /**
+      Component's wrapping <div> CSS-classes names.
+
+      Any other CSS-classes can be added through component's 'class' property.
+      ```handlebars
+      {{flexberry-icon
+        class="map icon"
+      }}
+      ```
+
+      @property classNames
+      @type String[]
+      @default ['flexberry-icon']
```

Comment:
does not correspond reality, should be ['flexberry-colorpicker']. And some other comments are from flexberry-ddau-checkbox